README.md

```diff
@@ -22,39 +22,33 @@ This repository contains tooling to deploy autonomous service agent(s) on Amazon
 
 After the deployment process finishes, the agent will be running in a [`screen`](https://www.gnu.org/software/screen/) session within an [AWS EC2](https://aws.amazon.com/ec2/) instance in the default AWS Region `us-east-2`.[^1]
 
-[^1]: If you wish to deploy on another AWS Region, you need to modify the Terraform variable `deployment_region` in the file `./infra/aws/docker_compose/variables.tf`. You also need to provide a valid Amazon Machine Image (AMI) ID for that region (resource `aws_instance` in the file `./infra/aws/docker_compose/main.tf`), otherwise the deployment process will fail on `terraform apply`.
-
 > **Note** <br />
-> **This repository contains default configuration parameters for the Autonomous Keeper Service (AKS) in the `./config/service_vars.env` file. To deploy this particular service, you will also need to set these variables:**
->
-> - **`SERVICE_REPO_URL`: <https://github.com/valory-xyz/agent-academy-2>**
-> - **`SERVICE_ID`: `valory/keep3r_bot_goerli:0.1.0` (for Görli testnet) or `valory/keep3r_bot:0.1.0` (for Ethereum mainnet)**
-> - **(Optional) `SERVICE_REPO_TAG`: `v0.3.0`**
->
-> **See the complete instructions below.**
+> **This repository contains default configuration parameters in the `./config/service_vars.env` file that you will need to override with the information you will have been provided by the service owner.**
+
+[^1]: If you wish to deploy on another AWS Region, you need to modify the Terraform variable `deployment_region` in the file `./infra/aws/docker_compose/variables.tf`. You also need to provide a valid Amazon Machine Image (AMI) ID for that region (resource `aws_instance` in the file `./infra/aws/docker_compose/main.tf`), otherwise the deployment process will fail on `terraform apply`.
 
 ## Prerequisites
 
 1. **Set up your AWS account.** Sign in to the AWS Management Console and configure the following parameters.
 
-   1. In case you don't have one, you need to create an IAM user with an access key. Within the AWS Management Console, create a new user (IAM/Users), and [create an access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for that user (Security credentials/Access keys). Note down the *AWS Access Key ID* and *AWS Secret Access Key*.
+   1. In case you don't have one, you need to create an IAM user with an access key. Within the AWS Management Console, create a new user (IAM/Users), and [create an access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for that user (Security credentials/Access keys). The IAM user needs have the permissions `AmazonEC2FullAccess`, `AmazonS3FullAccess` and `IAMFullAccess`. Note down the *AWS Access Key ID* and *AWS Secret Access Key*.
```

Comment:
I think IAMFullAccess is not required, but AmazonVPCFullAccess maybe is required.